src/main/java/service/impl/OperationProcessorImpl.java

```diff
@@ -0,0 +1,16 @@
+package service.impl;
+
+import java.util.List;
+import model.FruitTransaction;
+import service.OperationStrategy;
+import service.operation.OperationProcessor;
+
+public class OperationProcessorImpl implements OperationProcessor {
+    @Override
+    public void processConvertedData(List<FruitTransaction> fruitTransactionList,
+                                     OperationStrategy operationStrategy) {

```

Comment:
```suggestion
OperationStrategy operationStrategy;

public OperationProcessorImpl(OperationStrategy operationStrategy) {
this.operationStrategy = operationStrategy;
}
    @Override
    public void processConvertedData(List<FruitTransaction> fruitTransactionList) {
```
Let’s  pass  operationStrategy in constructor